controllers/oneagent/config.go

```diff
@@ -0,0 +1,9 @@
+package oneagent
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/logger"
+)
+
+var (
+	log = logger.NewDTLogger().WithName("dynakube-oneagent")
```

Comment:
`oneagent-controller`?

Reply:
same as before, its not a controller, and its called from the `dynakube-controller`